lang/cs/Org.Apache.REEF.Client/AzureBatch/AzureBatchDotNetClient.cs

```diff
@@ -112,15 +111,14 @@ private string GetCommand(JobParameters jobParameters)
 
             var command = azureBatchJobCommandBuilder.BuildDriverCommand(jobParameters.DriverMemoryInMB);
 
-            Log.Log(Level.Verbose, "Command for Azure Batch: {0}", command);
             return command;
         }
 
         public IJobSubmissionResult SubmitAndGetJobStatus(JobRequest jobRequest)
         {
             Submit(jobRequest);
 
-            // TODO[fix this null]
+            // TODO[Figure out how to return result from AzureBatch]
```

Comment:
Plan to leave it as this. I checked the implementation of local, YARN - the idea is that driver is running in the http server and this is used to talk to driver though Url. 

What AzureBatch does to get the task status by pulling could be implemented in GetJobFinalStatus(string jobId) (Line 73). Any comments?

Reply:
My concern is if we put a dummy implementation, it will still calling to DriverUrl uing Http, which is actually blocking for something we know it won't work for sure.

FetchDriverStatus() in JobSubmissionResult.cs

Reply:
Or you mean implementing IJobSubmissionResult instead of JobSubmissionResult?